docs/Oracle Jobs/jobs.md

```diff
@@ -0,0 +1,272 @@
+- `keyBundleID`: the hash of the OCR key bundle to be used by this node (the Chainlink node keystore manages these key bundles — use the subcommands under `chainlink keys ocr` to interact with them or use the Operator UI).
+- `monitoringEndpoint`: the URL of the telemetry endpoint to send OCR metrics to.
+- `transmitterAddress`: the Ethereum address from which to send aggregated submissions to the OCR contract.
+- `observationTimeout`: the maximum duration to wait before an off-chain request for data is considered to be failed/unfulfillable.
+- `blockchainTimeout`: the maximum duration to wait before an on-chain request for data is considered to be failed/unfulfillable.
+- `contractConfigTrackerSubscribeInterval`: the interval at which to retry subscribing to on-chain config changes if a subscription has not yet successfully been made.
+- `contractConfigTrackerPollInterval`: the interval at which to proactively poll the on-chain config for changes.
+- `contractConfigConfirmations`: the number of blocks to wait after an on-chain config change before considering it worthy of acting upon.
+
+### VRF
+
+```toml
+uuid = "123e4567-e89b-12d3-a456-426655440000"
+type = "randomnesslog"
+schemaVersion = 1
+name = "vrf-primary"
+coordinatorAddress = "0xaba5edc1a551e55b1a570c0e1f1055e5be11eca7"
+confirmations = 6
+publicKey = "0x79BE667EF9DCBBAC55A06295CE870B07029BFCDB2DCE28D959F2815B16F8179800"
+observationSource="""
+getrandomvalue [type=vrf];
+"""
+```
+
+- `uuid`: the globally-unique ID of the job. User-specified or randomly-generated.
+- `coordinatorAddress`: the address of the VRFCoordinator contract coordinating requests and responses for randomness.
+- `confirmations`: the number of blocks to wait before a request is considered final enough to be worth responding to.
+- `publicKey`: the public key of the VRF keypair securing the randomness response.
+
+---
+
+## Writing pipelines
+
+Pipelines are composed of tasks arranged in a DAG (directed acyclic graph) expressed in DOT syntax:
+
+[DOT (graph description language) - Wikipedia](https://en.wikipedia.org/wiki/DOT_%28graph_description_language%29#Directed_graphs)
+
+Each node in the graph is a task, and has both a user-specified ID as well as a set of configuration parameters/attributes:
+
+```dot
+my_fetch_task [type="http" method="get" url="https://chain.link/eth_usd"]
+```
+
+The edges between tasks define how data flows from one task to the next. Some tasks can have multiple inputs (such as `median`), while others are limited to 0 (`http`) or 1 (`jsonparse`).
+
+```dot
+data_source_1  [type="http" method="get" url="https://chain.link/eth_usd"]
+data_source_2  [type="http" method="get" url="https://coingecko.com/eth_usd"]
+medianize_data [type="median"]
+submit_to_ea   [type="bridge" name="my_bridge"]
+
+data_source_1 -> medianize_data
+data_source_2 -> medianize_data
+medianize_data -> submit_to_ea
+```
+
+![DAG Example](./dag_example.png)
```

Comment:
This image isn't showing on my local or the generated preview. I believe assets like this are stored in `_src/images/`.

Reply:
Good catch

Reply:
I think the image still doesn't show up